guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -8,6 +8,9 @@ Optionally, you can also bind the disk to a Tang server through Clevis for decry
 Associate the partition table with your operating system entry.
 Then, you assign the partition table to your host group or select it manually during provisioning.
 
+.Prerequisites
+* Ensure that your host has access to the `{client-clevis-repository}` repository to install `clevis` during the provisioning process.
```

Comment:
```suggestion
* Ensure that your host has access to the `{client-clevis-repository}` repository to install `clevis` during provisioning.
```
or even
```suggestion
* Your host has access to the `{client-clevis-repository}` repository to install `clevis` during provisioning.
```

Reply:
Applied the second suggestion which reads well!